README.md

```diff
@@ -129,6 +129,98 @@ The image ships with some basic fonts (`font-terminus font-inconsolata font-deja
 - Alpine Linux see https://wiki.alpinelinux.org/wiki/Fonts
 - Debian/Ubuntu see https://wiki.debian.org/Fonts
 
+#### Custom fonts directory
+
+Let's say your building a docker image for you application that requires the `ffmpeg` and `ffprobe` binaries, and  want to have a dedicated directory for storing fonts, e.g. `/app/fonts`. In order to achieve this, let's take a look at the following steps.
+
+1. A custom config file containing the fonts directory you want use has to be added to the image. Here, a file `50-custom.conf` will be added to the `/etc/fonts/conf.d` directory, containing an additional `/app/fonts` directory for Fontconfig to use.
+
+```Dockerfile
+COPY <<EOF /etc/fonts/conf.d/50-custom.conf
```

Comment:
`COPY` will create directories if they don't exist or this depend on base image has them?

Reply:
The parent directories are created if they don't exist according to docs: https://github.com/moby/buildkit/blob/2ec1338fc13f73b43f0b1b4f4678d7cd654bc86c/frontend/dockerfile/docs/reference.md?plain=1#L1458